src/remotestorage.ts

```diff
@@ -368,7 +370,7 @@ export class RemoteStorage {
     if (typeof cfg === 'object') { extend(config, cfg); }
 
     this.addEvents([
-      'ready', 'authing', 'connecting', 'connected', 'disconnected',
+      'ready', 'authing', 'connecting', 'pod-not-selected', 'connected', 'disconnected',
```

Comment:
... because no existing app will respond to the 'pod-not-selected' event, but they will respond to other events.

Reply:
This would be the state of the connection for the widget or someone implementing custom UI. So it should be part of a documentation page for adding Solid to an app (maybe another section on the GDrive/Dropbox page, and renaming the page and menu item).